### ISSUE TITLE
source code from demo

### ISSUE THREAD

**Issue by danielkalfas** [wittlock/ngx-image-zoom#37]
Hello,

is it possible to publish the source code from the demo "https://wittlock.github.io/ngx-image-zoom/" ?

**Comment by PhilippMeissner** [comment]
Yeah, I would second that :)
Or at least provide the actual code used in each scenario - Would be quite helpful for easier c+p.

**Comment by wittlock** [comment]
@danielkalfas Yeah, I don't see why not. I'll try to get that sorted in the near future. Didn't really think of it as a good source of inspiration but I'll get that into this repository and update the readme and demo page with a link.

**Comment by Tweniee** [comment]
@wittlock This option proves beneficial for developers as it provides clarity on the elements to be utilized in both HTML and TypeScript component files. Additionally, it offers a visual representation of its functionality. To enhance user interaction, consider incorporating dynamic settings, allowing users to customize aspects like lens size (width and height). This suggestion is based on observing similar features in various applications.

> @danielkalfas Yeah, I don't see why not. I'll try to get that sorted in the near future. Didn't really think of it as a good source of inspiration but I'll get that into this repository and update the readme and demo page with a link.
